nodejs/src/utils.ts

```diff
@@ -74,6 +82,25 @@ export async function writeConfig(baseDir: string = process.cwd()) {
     await writeConfigTemplates(kapetaVariables, baseDir);
 }
 
+
+export async function getEnvironmentVariables(kapetaVariables:Variables): Promise<Record<string, string>> {
+    const env: Record<string, string> = {};
+    for (const key in kapetaVariables) {
+        const variable = kapetaVariables[key];
+
+        if (variable.type !== VariableType.MAPPED &&
+            variable.value.length > MAX_ENV_LENGTH)  {
+            // Skip large values except if they are specifically mapped
+            continue;
```

Comment:
This sounds tricky, when is a variable mapped vs not? 🤔

Reply:
User made a kapeta.config.env file where they've specifically defined a new variable to contain an existing one. We always want to provide them as env vars since that's what they've asked us to do